### ISSUE TITLE
Support for Jellyfin 10.7.0?

### ISSUE THREAD

**Issue by mattssn11** [oonqt/MBCord#19]
Hello, I have upgraded Jelly fin to 10.7.0 which is the beta release, but now seem unable to connect to my Jellyfin server on MBcord, is this beta not supported yet?

** Replace [] with [x] to signify you've completed the following

** [x] I have read over the setup guide (https://github.com/oonqt/MBCord/wiki/Initial-Setup)

** [x] I have read over the commonly asked questions (https://github.com/oonqt/MBCord/wiki/Commonly-Asked-Issues-Questions)

**Comment by oonqt** [comment]
I just tested with version 10.7.0 and it was working fine. Could you provide some log files from when it failed to connect?

**Comment by mattssn11** [comment]
HI, I am just receiving [Mon Jan 25 2021 18:58:31 GMT-0700 (Mountain Standard Time)] error: Status: 404 Response: undefined.

I am using the same options as before, I chose https and I used port 443, but this no longer works

**Comment by mattssn11** [comment]
Now I am getting 

[Mon Jan 25 2021 19:13:23 GMT-0700 (Mountain Standard Time)] error: {
  "errno": "ECONNREFUSED",
  "code": "ECONNREFUSED",
  "syscall": "connect",
  "address": "x.x.x.x",
  "port": 8096

**Comment by oonqt** [comment]
Are you able to connect to the Demo Jellyfin server? https://demo.jellyfin.org/unstable port 443? This is the server that I used to test the beta was working properly.

**Comment by fringillidaes** [comment]
Can confirm this is happening to me too on 10.7 RC-2.

**Comment by mattssn11** [comment]
I have tried with the demo, using:

Https://
Host: https://demo.jellyfin.org/unstable
Port: 443
User: Demo
Pass: NONE

![nW5M1rz](https://user-images.githubusercontent.com/54347017/105815722-9a9eb400-5f70-11eb-845c-b545e958cd06.png)

**Comment by mattssn11** [comment]
I just tried with https://demo.jellyfin.org (Not Unstable) and I am not able to connect to that one.

**Comment by oonqt** [comment]
> Can confirm this is happening to me too on 10.7 RC-2. I can spin up a test server if needed to debug.

Could you please try going back to stable, check it's not happening over there, and go back to RC-2 to confirm it's not working?
I'm wondering if they broke something in RC-2 and resolve it in RC-3 which is what I assume the unstable demo is using.

**Comment by oonqt** [comment]
Actually, cancel that. I spun up a docker instance with 10.7 today and was able to track down the issue. What is going on is that Jellyfin changed how URLs are parsed in the 10.7 betas which would result in 404s being thrown if a URL contained multiple forward slashes (ie. http://x.x.x.x:8096//Users/authenticatebyname would result in a 404 in 10.7 RC-x, but not in 10.6. some of the urls). I did not notice this until now, but with the way I build URLs for http requests results in an extra slash being added to the very end of the server address, resulting in all URLs being built with double slashes.  I have published this release which resolves the issues with the URL paths: [https://github.com/oonqt/MBCord/releases/tag/2.1.11](https://github.com/oonqt/MBCord/releases/tag/2.1.11)
